### PR TITLE
feat: issue TLS-server certificates

### PR DESCRIPTION
**Pull request by emjunior258**
Changes
- Issue TLS server certificates
- Fix certeasy-core failing tests
- add organization & organization unit to issuers

### CODE REVIEW

File: certeasy-backend-app/src/main/java/org/certeasy/backend/common/OrganizationInfo.java [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+import java.util.HashSet;
+import java.util.Set;
+
+public record OrganizationInfo (
+
+    @JsonProperty("organization_name")
+    String organizationName,
+
+    @JsonProperty("organization_unit")
+    String organizationUnit) implements Validable {
```
Comment (snhane) [inline certeasy-backend-app/src/main/java/org/certeasy/backend/common/OrganizationInfo.java:18]: Remove the bracket closure

File: certeasy-backend-app/src/main/java/org/certeasy/backend/persistence/AbstractIssuerRegistry.java [hunk 2]
```diff
@@ -82,4 +82,9 @@ public void delete(CertIssuer issuer) throws IssuerRegistryException {
     }
 
 
+    protected void emptyChildrenMap() {
+        this.childrenMap.clear();;
```
Comment (snhane) [inline certeasy-backend-app/src/main/java/org/certeasy/backend/persistence/AbstractIssuerRegistry.java]: Double colon?

**Review by snhane — APPROVED** [review]
@emjunior258 Please review the 2 comments left and only then merge

File: certeasy-backend-app/src/main/java/org/certeasy/backend/common/OrganizationInfo.java (hunk 1 above)
Comment (emjunior258) [reply certeasy-backend-app/src/main/java/org/certeasy/backend/common/OrganizationInfo.java:18]: I don't understand. Which bracket closure @snhane ?

File: certeasy-backend-app/src/main/java/org/certeasy/backend/persistence/AbstractIssuerRegistry.java (hunk 2 above)
Comment (emjunior258) [reply certeasy-backend-app/src/main/java/org/certeasy/backend/persistence/AbstractIssuerRegistry.java]: Fixed it

File: certeasy-backend-app/src/main/java/org/certeasy/backend/common/OrganizationInfo.java (hunk 1 above)
Comment (snhane) [reply certeasy-backend-app/src/main/java/org/certeasy/backend/common/OrganizationInfo.java:18]: Line 18: organizationUnit**)**

Comment (snhane) [reply certeasy-backend-app/src/main/java/org/certeasy/backend/common/OrganizationInfo.java:18]: > I don't understand. Which bracket closure @snhane ?

Got it. 

My rainbow brackets are failing me

**Review by snhane — APPROVED** [review]
clear

**Pull request merged**